mesonbuild/backend/xcodebackend.py

```diff
@@ -136,6 +136,9 @@ def add_item(self, key, value, comment=''):
         self.keys.add(key)
         self.items.append(item)
 
+    def has_item(self, key):
+        return key in self.keys
```

Comment:
This isn't very idiomatic python. Why not implement a `__contains__` dunder method?